tests/ValidatorTest.php

```diff
@@ -338,4 +339,22 @@ public function testGetValidRuleProperties()
     {
         $this->assertContains('required', $this->validator->getValidRuleProperties());
     }
+
+    public function testValidateNullableValueDoesNotReturnErrorMsg()
```

Comment:
Also add tests for sending in other data here - like strings, `0`, empty strings, etc to make sure this always work

Reply:
good idea!